cdap-docs/admin-manual/incompatibilities.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2019 Cask Data, Inc.
+
+.. _admin-manual-incompatibilities:
+
+=============================
+Incompatibility with CDAP 5.x
+=============================
+
+CDAP 6.0.0 has several incompatible changes, because of that CDAP 6.0.0 does not support upgrade or migration from
+CDAP 5.x to 6.0.0. Below is the list of incompatibilities:
+
+- **CDAP Applications and Custom Plugins:**
+   * In CDAP 6.0.0, cdap packages have been renamed from `co.cask.cdap` to `io.cdap.cdap`. So all the CDAP applications
+   and custom plugins must be updated and recompiled before using them in CDAP 6.0.0.
+   * In CDAP 6.0.0, support for CDAP Flows and Streams have been removed. As an alternate, CDAP applications can rely on
+    Spark Streaming.
+- **CDAP Pipelines:**
+   * All the pipelines from existing CDAP cluster must be exported and re-imported into CDAP 6.0.0 cluster.
```

Comment:
Change to:
All pipelines from existing CDAP clusters must be exported and re-imported into CDAP 6.0.0 clusters.